python/pyspark/pandas/window.py

```diff
@@ -706,6 +706,10 @@ def _apply_as_series_or_frame(self, func: Callable[[Column], Column]) -> FrameLi
         if groupby._agg_columns_selected:
             agg_columns = groupby._agg_columns
         else:
+            # pandas doesn't keep the groupkey as a column from 1.3 for DataFrameGroupBy
+            if isinstance(groupby, DataFrameGroupBy):
+                for groupkey in groupby._groupkeys:  # type: ignore
+                    groupby._column_labels_to_exlcude.add(groupkey._internal.column_labels[0])
```

Comment:
Should we rename the from `exlcude` to `exclude` in this PR, too. ?? It's typo.

Reply:
yeah please go ahead.